aws-android-sdk-connect/src/main/java/com/amazonaws/services/connect/model/PropertyValidationExceptionProperty.java

```diff
@@ -0,0 +1,308 @@
+     * </p>
+     * <p>
+     * Returns a reference to this object so that method calls can be chained
+     * together.
+     *
+     * @param propertyPath <p>
+     *            The full property path.
+     *            </p>
+     * @return A reference to this updated object so that method calls can be
+     *         chained together.
+     */
+    public PropertyValidationExceptionProperty withPropertyPath(String propertyPath) {
+        this.propertyPath = propertyPath;
+        return this;
+    }
+
+    /**
+     * <p>
+     * Why the property is not valid.
+     * </p>
+     * <p>
+     * <b>Constraints:</b><br/>
+     * <b>Allowed Values: </b>INVALID_FORMAT, UNIQUE_CONSTRAINT_VIOLATED,
+     * REFERENCED_RESOURCE_NOT_FOUND, RESOURCE_NAME_ALREADY_EXISTS,
+     * REQUIRED_PROPERTY_MISSING, NOT_SUPPORTED
+     *
+     * @return <p>
+     *         Why the property is not valid.
+     *         </p>
+     * @see PropertyValidationExceptionReason
+     */
+    public String getReason() {
+        return reason;
+    }
+
+    /**
+     * <p>
+     * Why the property is not valid.
+     * </p>
+     * <p>
+     * <b>Constraints:</b><br/>
+     * <b>Allowed Values: </b>INVALID_FORMAT, UNIQUE_CONSTRAINT_VIOLATED,
+     * REFERENCED_RESOURCE_NOT_FOUND, RESOURCE_NAME_ALREADY_EXISTS,
+     * REQUIRED_PROPERTY_MISSING, NOT_SUPPORTED
+     *
+     * @param reason <p>
+     *            Why the property is not valid.
+     *            </p>
+     * @see PropertyValidationExceptionReason
+     */
+    public void setReason(String reason) {
+        this.reason = reason;
+    }
+
+    /**
+     * <p>
+     * Why the property is not valid.
+     * </p>
+     * <p>
+     * Returns a reference to this object so that method calls can be chained
+     * together.
+     * <p>
+     * <b>Constraints:</b><br/>
+     * <b>Allowed Values: </b>INVALID_FORMAT, UNIQUE_CONSTRAINT_VIOLATED,
+     * REFERENCED_RESOURCE_NOT_FOUND, RESOURCE_NAME_ALREADY_EXISTS,
+     * REQUIRED_PROPERTY_MISSING, NOT_SUPPORTED
+     *
+     * @param reason <p>
+     *            Why the property is not valid.
+     *            </p>
+     * @return A reference to this updated object so that method calls can be
+     *         chained together.
+     * @see PropertyValidationExceptionReason
+     */
+    public PropertyValidationExceptionProperty withReason(String reason) {
+        this.reason = reason;
+        return this;
+    }
+
+    /**
+     * <p>
+     * Why the property is not valid.
+     * </p>
+     * <p>
+     * <b>Constraints:</b><br/>
+     * <b>Allowed Values: </b>INVALID_FORMAT, UNIQUE_CONSTRAINT_VIOLATED,
+     * REFERENCED_RESOURCE_NOT_FOUND, RESOURCE_NAME_ALREADY_EXISTS,
+     * REQUIRED_PROPERTY_MISSING, NOT_SUPPORTED
+     *
+     * @param reason <p>
+     *            Why the property is not valid.
+     *            </p>
+     * @see PropertyValidationExceptionReason
+     */
+    public void setReason(PropertyValidationExceptionReason reason) {
```

Comment:
The definition of `PropertyValidationExceptionReason` is probably missing which is failing the build, right?